src/components/Post.js

```diff
@@ -0,0 +1,85 @@
+import { useState } from "react";
+import axiosInstance from "../api/axios";
+import img1 from "../images/img1.jpg";
+import { useNavigate } from "react-router-dom";
+
+
+const Post = () => {
+  const [blog, setBlog] = useState({ title: "", content: "", author: "" });
+  let navigate = useNavigate();
+
+  const getBlogDetails = (e) => {
+    e.preventDefault();
+    setBlog((current) => ({ ...current, [e.target.name]: e.target.value }));
```

Comment:
`current` or `previous`?

Reply:
Don't you think `current` is a bad name for something that is `previous`

Reply:
oh mine!, I thought I have changed this since.